edx/edx-app-ios
# OSPR-2505 Set correct tint color for wifi switch

### Description

[OSPR-2505](https://openedx.atlassian.net/browse/OSPR-2505)

This PR shows the `primary color` on the `wifi switch button` instead of the default one from the `OEXMySettingsViewController.xib` file. 

### How to test this PR

Change the primary color to a new value. Run the app. Go to Account / Settings. Turn on and off the switch and see that it is using the primary color.

### Reviewers

cc @marcotuts @staubina 

![](https://media.giphy.com/media/ruw1bRYN0IXNS/giphy.gif)
